montecristo/src/main/kotlin/com/datastax/montecristo/model/versions/dse/DseV69X.kt

```diff
@@ -0,0 +1,34 @@
+package com.datastax.montecristo.model.versions.dse
+
+import com.datastax.montecristo.logs.LogEntry
+import com.datastax.montecristo.logs.Searcher
+import com.datastax.montecristo.logs.logMessageParsers.DseTombstoneWarningMessage
+import com.datastax.montecristo.logs.logMessageParsers.TombstoneWarningMessage
+import com.datastax.montecristo.model.logs.LogLevel
+import com.datastax.montecristo.model.versions.DatabaseVersion
+
+class DseV69X(versionIdentifier: String) : Dse6Base(versionIdentifier) {
+    override fun supportsReadRepair(): Boolean {
+        return false
+    }
+    override fun isCommunityMaintained(): Boolean {
```

Comment:
DSE 6.9 is classed as community maintained?

Reply:
idk 🤷
all the dse versions are.  i just assume it's the equivalent, i.e. "datastax supported"
https://github.com/datastax-labs/Montecristo/blob/main/montecristo/src/main/kotlin/com/datastax/montecristo/model/versions/dse/DseV68X.kt#L31

Reply:
renamed the method to `isSupported()`